numpy/core/fromnumeric.py

```diff
@@ -797,6 +797,7 @@ def argpartition(a, kth, axis=-1, kind='introselect', order=None):
     partition : Describes partition algorithms used.
     ndarray.partition : Inplace partition.
     argsort : Full indirect sort
+    take_along_axis : Take values from the input array by matching 1d index and data slices.
```

Comment:
Let's see how another maintainer feels about this first, but perhaps:

```suggestion
    take_along_axis : Take values from the input array by matching 1d index and data slices.
    take_along_axis : Apply ``index_array`` from argpartition to an array as if by calling partition
```

And same for the below,
```
    take_along_axis : Apply ``index_array`` from argsort to an array as if by calling sort
    take_along_axis : Apply ``np.expand_dims(index_array, axis)`` from argmin to an array as if by calling min
    take_along_axis : Apply ``np.expand_dims(index_array, axis)`` the indices from argmax to an array as if by calling max
```

Reply:
Probably those use cases are worth mentioning, but I wasn't sure if the "See also" section is a right place for that. e.g. argsort and argpartition refer to take_along_axis in "Returns". Perhaps it's better to just add an example.

Reply:
An example for each of these showing how to use `take_along_axis` would be great, if you're willing to spend the time. If not, we can merge this as is, and leave the issue open for someone else to add the examples.

Reply:
"See Also" seems appropriate here.